### DISCUSSION TITLE
Media section for a user's page

### DISCUSSION THREAD

**Discussion by Zero3K** [bluesky-social/atproto#1345] [Protocol (atproto)]
It would be nice if there was a Media section for a user's page on Bluesky.

**Comment by Zero3K** [comment]
Any thoughts on this?

**Reply by bnewbold** [reply]
Hey @Zero3K, a number of folks have requested this feature and it is on our list of product/feature ideas. We don't have any timeline for this specific feature.

**Reply by Zero3K** [reply]
A third-party web-based app has already managed to add it. Maybe you can use their code. The source code of it can be found at https://github.com/callmearta/kite.

**Comment by Zero3K** [comment]
The requested feature has been added. So, I am going to close this discussion.
